lesson03/script.js

```diff
@@ -0,0 +1,17 @@
+const ul = document.getElementById('js-list');
+const aHref = ['a1.html','a2.html'];
+const aText = ['a1','a2'];
+const imgSrc = ['./img/bookmark.png','./img/message.png'];
+
+for (let i= 0; i < aHref.length; i++){
+    const li = document.createElement('li');
+
+    const a = document.createElement('a');
+    a.href = aHref[i];
+    a.textContent = aText[i];
```

Comment:
IMO

I think you can wright with "Template literals" here as below. Then the arrays "const aHref" and "const aText" are no longer needed.

Reference:
https://www.javadrive.jp/javascript/string/index5.html

```suggestion
    a.href = `a${i+1}.html`;
    a.textContent = `a${i+1}`;
```


or If you set the initial value here from 1, you won't need "+1".
For example,

``for (let i= 1; i <= liLength; i++){...``
``    a.href = `a${i}.html`;``
 ``   a.textContent = `a${i}`;``

Reply:
I don't know "Template literals".
I will study, and challenge to write with  "Template literals".